blueprints/codes/eurocode/nen_en_1993_1_9_c2_2012/annex_a_determination_of_fatigue_load_parameters_and_verification_formats/formula_a_2.py

```diff
@@ -0,0 +1,52 @@
+"""Formula A.1 from NEN-EN 1993-1-9+C2:2012: Annex A - Determination of fatigue load parameters and verification formats."""
+
+from blueprints.codes.eurocode.nen_en_1993_1_9_c2_2012 import NEN_EN_1993_1_9_C2_2012
+from blueprints.codes.formula import Formula
+from blueprints.codes.latex_formula import LatexFormula
+from blueprints.type_alias import DIMENSIONLESS
+from blueprints.validations import raise_if_negative
+
+
+class FormADot2CriteriaBasedOnDamageAccumulation(Formula):
+    """Class representing formula A.2 for the calculation of the damage during the design life."""
+
+    label = "A.2"
+    source_document = NEN_EN_1993_1_9_C2_2012
+
+    def __init__(
+        self,
+        d_d: DIMENSIONLESS,
+    ) -> None:
+        """[:math:`OK?`] Criteria met, based on damage accumulation.
+
+        NEN-EN 1993-1-9+C2:2012 art.A.5 - Formula (A.1)
+
+        Parameters
+        ----------
+        d_d : DIMENSIONLESS
+            [:math:`D_d`] The damage during the design life [:math:`-`].
+
+        Returns
+        -------
+        None
+        """
+        super().__init__()
+        self.d_d = d_d
+
+    @staticmethod
+    def _evaluate(
+        d_d: DIMENSIONLESS,
+    ) -> bool:
```

Comment:
Geen Bool maar string gebruiken. 'OK' of 'NOT OK'. Of een Enumerate of een dictionary

Reply:
Is not possible in current architecture. Proposal to keep format